src/parser/parser_utils.js

```diff
@@ -202,7 +202,34 @@ function mergeWrapperAdData(unwrappedAd, wrapper) {
   );
   unwrappedAd.extensions = wrapper.extensions.concat(unwrappedAd.extensions);
 
+  const wrapperCompanions = (wrapper.creatives || []).filter(
+    creative => creative && creative.type === 'companion'
+  );
+  const wrapperCompanionClickTracking = wrapperCompanions.reduce(
+    (result, creative) => {
+      (creative.variations || []).forEach(variation => {
```

Comment:
Not sure this check really fits our safety needs, what if `creative.creations` is an object?
```javascript
({} || []).forEach(() => {}) // forEach is not a function
```

Reply:
variations is always an array

Reply:
then `creative.variations` will always vslidate to `true` and we don't need the safety check no?

Reply:
I meant that it is never an object, but AFAIK it can be either array, null or undefined